android/src/main/java/tech/bam/RNBatchPush/RNBatchModule.java

```diff
@@ -141,6 +142,11 @@ public void optOutAndWipeData(Promise promise) {
     private void sendEvent(ReactContext reactContext,
                            String eventName,
                            @Nullable WritableMap params) {
+
+        if (!reactContext.hasActiveCatalystInstance()) {
+            Log.d(NAME, "React context has no active catalyst instance. Aborting send event.");
+            return;
+        }
         reactContext
                 .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
                 .emit(eventName, params);
```

Comment:
Does it crash here? If so there might be a bug in RN and we should open an issue on their repo